apps/emqx_mt/src/emqx_mt_state.erl

```diff
@@ -446,6 +452,7 @@ create_managed_ns_txn(Ns) ->
     end.
 
 delete_managed_ns_txn(Ns) ->
```

Comment:
rename the function to `delete_ns_txn` ?

Reply:
:checkered_flag: